.github/workflows/main.yml

```diff
@@ -55,6 +61,12 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Update DotNet SDK version
+        run: |
+          version=$(grep -o '"version": "[^"]*"' libs/dotnet-sdk/package.json | awk -F'"' '{print $4}')
```

Comment:
Cross platform way to do this

`node -pe JSON.stringify({ version: require('libs/dotnet-sdk/package.json').version })`